akka-javasdk/src/main/scala/akka/javasdk/impl/agent/AgentImpl.scala

```diff
@@ -129,6 +129,8 @@ private[impl] final class AgentImpl[A <: Agent](
               systemMessage,
               req.userMessage,
               req.responseType,
+              req.responseMapping,
```

Comment:
Not sure if passing back and forth is a good idea, but I don't see any other way